sormas-ui/src/main/java/de/symeda/sormas/ui/user/UserEditForm.java

```diff
@@ -220,15 +221,15 @@ private void updateFieldsByUserRole() {
 		}
 
 		ComboBox region = (ComboBox) getFieldGroup().getField(UserDto.REGION);
-		boolean useRegion = isSupervisor || isInformant || isOfficer || isStateObserver || isDistrictObserver;
+		boolean useRegion = jurisdictionLevel == JurisdictionLevel.REGION || isInformant || jurisdictionLevel == JurisdictionLevel.DISTRICT;
 		region.setVisible(useRegion);
 		setRequired(useRegion, UserDto.REGION);
 		if (!useRegion) {
 			region.clear();
 		}
 
 		ComboBox district = (ComboBox) getFieldGroup().getField(UserDto.DISTRICT);
-		boolean useDistrict = isInformant || isOfficer || isDistrictObserver;
+		boolean useDistrict = isInformant || jurisdictionLevel == JurisdictionLevel.DISTRICT;
```

Comment:
Additional options are: JursidictionLevel.COMMUNITY and health facility and point of entry.